routes/authors.js

```diff
@@ -1,8 +1,55 @@
 const express = require("express");
 const generateId = require("uuid/v4");
+const authors = require("../data/authors");
+const body = require('body-parser')
```

Comment:
I don't think you need `body-parser` here.

Reply:
Was sending data in the wrong format, it was one of the things I tried, forgot to remove it